.ci/build-mac.sh

```diff
@@ -4,18 +4,21 @@ brew install -f --overwrite nasm ninja git p7zip create-dmg ccache pipenv
 
 #/usr/sbin/softwareupdate --install-rosetta --agree-to-license
 arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-arch -x86_64 /usr/local/homebrew/bin/brew update
-arch -x86_64 /usr/local/homebrew/bin/brew install -f --overwrite llvm@16 sdl2 glew cmake
-arch -x86_64 /usr/local/homebrew/bin/brew link -f llvm@16
+arch -x86_64 /usr/local/bin/brew update
+arch -x86_64 /usr/local/bin/brew install -f --overwrite llvm@16 sdl2 glew cmake faudio molten-vk@1.2.4 vulkan-headers
+arch -x86_64 /usr/local/bin/brew link -f llvm@16
 
+# moltenvk based on commit for 1.2.4 release
+wget https://raw.githubusercontent.com/Homebrew/homebrew-core/b233d4f9f40f26d81da11140defbfd578cfe4a69/Formula/molten-vk.rb
+arch -x86_64 /usr/local/bin/brew install -f --overwrite ./molten-vk.rb
```

Comment:
Why is molten-vk@1.2.4 installed above and then here as molten-vk.rb ?
What's the difference?

Reply:
i forgot to remove it hold on